src/RoleObjects/OrderedListData.test.js

```diff
@@ -0,0 +1,110 @@
+import * as createjs from 'createjs-module';
+import AccessibilityModule from '../index';
+import { parentEl, stage, container } from '../__jestSharedSetup';
+
+describe('OrderedListData', () => {
+  describe('register role', () => {
+    let cjsOrderedList;
+    let cjsCell;
+    let olEl;
+    let isReversed;
+    let startVal;
+    let typeVal;
+
+    beforeEach(() => {
+      cjsOrderedList = new createjs.Shape(); // dummy object
+      cjsCell = new createjs.Shape(); // dummy child object
+      isReversed = false;
+      startVal = 7;
+      typeVal = 'I';
+
+      AccessibilityModule.register({
+        accessibleOptions: {
+          reversed: isReversed,
+          start: startVal,
+          type: typeVal,
+        },
+        displayObject: cjsOrderedList,
+        parent: container,
+        role: AccessibilityModule.ROLES.ORDEREDLIST,
+      });
+
+      AccessibilityModule.register({
+        displayObject: cjsCell,
+        role: AccessibilityModule.ROLES.CELL,
+      });
+
+      stage.accessibilityTranslator.update();
+    });
+
+    describe('rendering', () => {
+      it('creates ol element', () => {
+        olEl = parentEl.querySelector('ol');
+        expect(olEl).not.toBeNull();
+      });
+    });
+
+    describe('children checking', () => {
+      const errorObj = /Children of orderedlist must have a role of listitem/;
+      it('throws error attempting to add prohibited child using addChild() ', () => {
+        expect(() => {
+          stage.accessibilityTranslator.update();
+          cjsOrderedList.accessible.addChild(cjsCell);
+        }).toThrowError(errorObj);
+      });
+
+      it('throws error attempting to add prohibited child using addChildAt()', () => {
+        expect(() => {
+          stage.accessibilityTranslator.update();
+          cjsOrderedList.accessible.addChildAt(cjsCell, 0);
+        }).toThrowError(errorObj);
+      });
+
+      const cjsDummy = new createjs.Shape();
+      AccessibilityModule.register({
+        displayObject: cjsDummy,
+        role: AccessibilityModule.ROLES.LISTITEM,
+      });
```

Comment:
What're these lines doing just hanging out? If they're needed for the next two cases you can just wrap the two cases in a describe, and throw them in a new `before`?